woocommerce-services.php

```diff
@@ -357,14 +357,17 @@ public function load_textdomain() {
 		 * @codeCoverageIgnore
 		 */
 		public function init() {
+			$this->load_dependencies();
 			add_action( 'admin_init', array( $this, 'admin_enqueue_scripts' ) );
 
 			if ( ! WC_Connect_Options::get_option( 'tos_accepted', false ) ) {
 				add_action( 'admin_init', array( $this, 'admin_tos_notice' ) );
 				return;
+			} else if ( ! $this->check_jetpack_install() ) {
```

Comment:
The problem with this approach is that the notice is never shown. I think you might be seeing it because perhaps you still have some leftover plugin option data?

With no WCS options in the `wp_options` table, and no Jetpack installed, this notice doesn't show.

See the `can_accept_tos()` function.